doc/articles/features/svg.md

```diff
@@ -12,6 +12,7 @@ Uno Platform supports using vector SVG graphics inside of your cross-platform ap
 
 To use SVG, install the following NuGet packages into the iOS, macOS, Mac Catalyst, Android, and Skia projects:
 
+* `Svg.Skia` (required in the Skia project)
```

Comment:
```suggestion
* `Svg.Skia` (required only for the Skia project)
```

Reply:
Also below if you can add an additional note please:
If the `Svg.Skia` package is not installed, you will get an error when an `.svg` image is loaded for the Skia project.
![image](https://github.com/unoplatform/uno/assets/16295702/ad3408de-90b5-4c69-a116-9f2d82449928)

Reply:
@mcNets here is the content you should have at the end:

```
## How to use SVG

To use SVG, install the following NuGet packages into the iOS, macOS, Mac Catalyst, Android, and Skia projects:

* `Uno.WinUI.Svg` (or `Uno.UI.Svg` if you are using a UWP-based app template)
* `SkiaSharp.Views.Uno.WinUI` (or `SkiaSharp.Views.Uno` if you are using a UWP-based app template)
* `Svg.Skia` (required only for the Skia project)

> [!NOTE]
> If the `Uno.[UI|WinUI].Svg` package is not installed, you will get a warning when an `.svg` image is loaded.
> If the `Svg.Skia` package is not installed, you will get an error when an `.svg` image is loaded for the Skia project.
> [!IMPORTANT]
> The `Uno.[UI|WinUI].Svg` package is not needed for WebAssembly, and must only be installed on the Mobile and Skia heads. It must not be in any other class libraries of your solution.
```

Reply:
I'm very surprised by this change. @mcNets did you get the build error reported here, when not having the Svg.Skia dependency: https://github.com/unoplatform/uno/blob/de215a6fb74e5976f27bc005564702cffc2471af/src/AddIns/Uno.UI.Svg/buildTransitive/Uno.UI.Svg.targets#L29 ? 

If not, it's not just a docs issue, and we'd need to fix the error reporting as well.

Reply:
@jeromelaban no , I didn't.  Should I close the PR and open a issue in regard of Uno.UI.Svg.targets?

Reply:
@jeromelaban when you are building the Skia project without the `Svg.Skia` package at the moment you properly get this error in the output (tested with the latest stable template + latest stable packages):
`6>C:\Users\Agnes Zitte\.nuget\packages\uno.winui.svg\5.1.87\buildTransitive\Uno.WinUI.Svg.targets(29,3): error : In order to use Uno Svg support, the 'Svg.Skia' NuGet Package (1.0.0.1 or later) must be referenced in the project. You can add <PackageReference Include="Svg.Skia" Version="1.0.0.1" /> to your project.`

@mcNets you already had the proper `Svg.Skia` package in your previous sample solutions for your tests so I don't think you noticed the error as the proper package was already there. But if you remove the package and try to build the Skia project you should see the error.

So I think we just need to update the doc only @jeromelaban

Reply:
Thanks @agneszitte!

@mcNets what was the original issue you were getting? The exact error?

Reply:
@jeromelaban The first error was this as Agnes pointed out:
```
 In order to use Uno Svg support, the 'Svg.Skia' NuGet Package (1.0.0.1 or later) must be referenced in the project. You can add <PackageReference Include="Svg.Skia" Version="1.0.0.1" /> to your project.
 ```
 
 I checked using a new project and still getting the same error.

Reply:
@jeromelaban The error only pops up after you add the .svg file to the project. Before that it compiles.

Reply:
@jeromelaban for more clarification, this is the issue @mcNets encountered on his side originally: https://github.com/unoplatform/uno/issues/15863

But while doing some tests with him for it, this is when I noticed we were missing details in the doc about the 'Svg.Skia' package for the Skia project and decided to open an issue for it (https://github.com/unoplatform/uno/issues/15864).

And @mcNets was very kind to offer his help in creating a PR in order to fix the documentation for it.

Reply:
@agneszitte I changed the file text. I was about to amend the previous commit's title but I'm not sure If I can use --force-with-lease in your repo.